app/services/follows/data_info.rb

```diff
@@ -0,0 +1,13 @@
+module Follows
+  module DataInfo
+    def self.to_json(object:, id: object.id, class_name: object.class.name, name: object.name, **kwargs)
+      kwargs.merge(
+        {
+          id: id,
+          className: class_name,
+          name: name
+        },
+      ).to_json
+    end
+  end
+end
```

Comment:
How does this module differ from the top-level one below?

Reply:
Gah, that creature came along for the ride.